developers/advanced/advanced-pinot-setup.md

```diff
@@ -311,6 +311,33 @@ bin/pinot-admin.sh AddTable \
 {% endtab %}
 {% endtabs %}
 
+### Automatically add an inverted index to your batch table
+
+It is possible to force the creation of an inverted index during segment generation by adding an entry to your [table index config](../../configuration-reference/table.md#table-index-config) in the table configuration file.
+
+This setting works with [batch (offline) tables](#batch-table-creation). This setting is `false` by default but you can set `createInvertedIndexDuringSegmentGeneration` to `true` in your table config, as follows:
+
+```json
+...
+"tableIndexConfig": {
+    ...
+    "createInvertedIndexDuringSegmentGeneration": true,
+    ...
+}
+...
+```
+
+When set to `true`, Pinot creates an inverted index for the columns that you specify in the `invertedIndexColumns` list in the table configuration. If `false`, no inverted index is created.
+
+When you update this setting in your table configuration, you must [reload the table segment](../../basics/data-import/segment-reload.md) to apply the inverted index to all existing segments.
+
+{% hint style="info" %}
+All other index types are created during the segment generation by default, and that is the preferred thing to do. However, since this is not the default for inverted indexes, this setting enables you to also create them at segment generation time.
```

Comment:
it sounds like we may be able to drop the word force....by default, the inverted index type is the only type of index that isn't created automatically during segment generation. To automatically create ....set to true